templates/server/index.html

```diff
@@ -326,11 +326,10 @@ <h2> Ubuntu Advantage for your Infrastructure </h2>
   </div>
   <div class="row">
     <div class="col-6">
-      <p> Ubuntu Advantage for infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry.
-        With OpenStack and Kubernetes support included, Ubuntu Advantage for Infrastructure provides everything you need to future-proof your data centre. </p>
-      <p> UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support and is available at Standard (24x5) and Advanced (24x7) response
-        times. </p>
-      <p><a href="/pricing/infra">Learn more about Ubuntu Advantage for Infrastructure&nbsp;&rsaquo;</a></p>
+      <p> Ubuntu Pro offers a single, per-node packaging of the most comprehensive software, hardening and security in the industry.
```

Comment:
@andreu thanks for this - all looks good except the title for this section also needs updating.

Currently it is: "Ubuntu Advantage for your Infrastructure"
and should just be "Ubuntu Pro"

Reply:
Done!